1-js/10-error-handling/1-try-catch/1-finally-or-code-after/solution.md

```diff
@@ -1,47 +1,47 @@
-The difference becomes obvious when we look at the code inside a function.
+Jeśli umieścimy pierwszy fragment w środku funkcji, szybko zauważymy różnicę. 
```

Comment:
W tłumaczeniu jest trochę zmienione znaczenie, "szybko", to nie to samo co "becomes obvious"